js/main.js

```diff
@@ -22,4 +22,67 @@ $form.addEventListener('submit', function (event) {
   data.entries.unshift(newObject);
   $imgSrc.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+  const $ul = document.querySelector('ul');
+  $ul.prepend(renderEntry(newObject));
+});
+
+function renderEntry(entry) {
+  const newLi = document.createElement('li');
+  const newDiv = document.createElement('div');
+  const newImg = document.createElement('img');
+  const newH2 = document.createElement('h2');
+  const newP = document.createElement('p');
+  newDiv.className = 'container';
+  newImg.setAttribute('src', entry.photoURL);
+  newLi.append(newDiv);
+  newDiv.append(newImg);
+  newImg.after(newH2);
+  newH2.after(newP);
+  newH2.textContent = entry.title;
+  newP.textContent = entry.notes;
```

Comment:
There should be a couple more elements here with some class names. We should be utilizing our layout classes that we provided for you to make this responsive. For example it should go `li` > `<div class="row">` > 2x `<div class="column-half">` > content goes inside these `div`s with column-half.

Reply:
Is this the proper HTML structure that I'm looking for? Just wanted to hardcode it and see what it looks like before I do it in the main.js file?

        <li>
          <div class="row">
            <div class="column-half">
              <!-- content goes here -->
              <div class="column-half">
                <!-- content goes here -->
              </div>
            </div>
          </div>
        </li>